types/dayjs-precise-range/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+      "npm-naming": false
```

Comment:
Yeah, don't disable this lint rule - it being off can mean the publisher fails to publish types.